cmd/influxd/run.go

```diff
@@ -198,7 +202,7 @@ func joinBroker(b *messaging.Broker, joinURLs []*url.URL) {
 }
 
 // creates and initializes a server.
-func openServer(path string, u *url.URL, b *messaging.Broker, initializing bool, joinURLs []*url.URL) *influxdb.Server {
+func openServer(path string, u *url.URL, b *messaging.Broker, initializing, configExists bool, joinURLs []*url.URL) *influxdb.Server {
```

Comment:
Minor: I do think this `configExists` logic should have been executed *before* the call to `openServer()`. That way `openServer()` remained untouched. Right now it seems a little off that `configExists` has to be pushed into the function.

I.e.

```
// Special case if no config file was specified.
if !configExists {
    joinURLs = []*url.URL{b.URL()}
}
.
.
.
s := openServer(config.Data.Dir, config.DataURL(), b, initializing, joinURLs)
```
This seems like clearer code to me.